modules/decima-ui/src/main/java/com/shade/decima/ui/data/viewer/texture/TextureViewer.java

```diff
@@ -28,11 +35,16 @@
 import java.util.Objects;
 import java.util.ServiceLoader;
 import java.util.stream.IntStream;
+import java.util.EnumSet;
 
 @ValueViewerRegistration({
-    @Type(name = "Texture", game = GameType.DS),
-    @Type(name = "Texture", game = GameType.DSDC),
-    @Type(name = "Texture", game = GameType.HZD),
+    @Type(name = "Texture", game = {GameType.DS, GameType.DSDC, GameType.HZD}),
+    @Type(name = "TextureSetEntry"),
+    @Type(name = "TextureBindingWithHandle"),
+    @Type(name = "UITexture"),
+    @Type(name = "TextureList"),
+    @Type(name = "StreamingRef<UITexture>"),
+    @Type(name = "Ref<UITexture>"),
```

Comment:
Support for references can be dropped now, I believe? There's now a special viewer for references that delegates to the target type's viewer.